### ISSUE TITLE
Remove BaseTabs

### ISSUE THREAD

**Issue by bennypowers** [patternfly/patternfly-elements#2624]
Affects `<rh-tab>`

See also #2577 
See also #

```[tasklist]
### Tasks
- [ ] https://github.com/patternfly/patternfly-elements/pull/2577
- [ ] https://github.com/patternfly/patternfly-elements/issues/2575
```

**Comment by bennypowers** [comment]
It's worth considering if we should shipg a separate AccordionController, or maybe modify TabsController to suit accordion as well,

or maybe have TabsController AND AccordionController, but have them both share code via a base class or a 3rd controller (remember: controllers are able to add other controllers)

**Comment by bennypowers** [comment]
Closed by #2745
